### DISCUSSION TITLE
How accurate the Pepy is?

### DISCUSSION THREAD

**Discussion by costis94** [psincraian/pepy#275] [Q&A] [answered]
Hi there, I loved your project!
But how accurate the download statistics is?

**Comment by psincraian** [comment, marked as answer]
Hey @costis94, well it depends on how do you want to mesure accuracy, is how many people uses it or how many downloads from PyPI it has?

If it's the last case I would say that is 100% accurate as we get the data from the [official BigQuery project](https://packaging.python.org/guides/analyzing-pypi-package-downloads/). But take into account that numbers can be higher or lower as described in the article.

**Reply by jonatan1609** [reply]
Well, why few projects which exist on Pypi does not exist on Pepy.tech?

**Reply by psincraian** [reply]
Hi @jonatan1609, the reason is because the project probably doesn't have any downloads. Do you have an example?

**Reply by jonatan1609** [reply]
> Hi @jonatan1609, the reason is because the project probably doesn't have any downloads. Do you have an example?

never mind and sorry for bothering, now I can see it.
